service/iot-service-client/src/main/java/com/microsoft/azure/sdk/iot/service/ServiceClient.java

```diff
@@ -19,7 +19,8 @@
 import java.util.concurrent.Executors;
 
 /**
- * Extend the ServiceClient class and provide AMPQ specific implementation.
```

Comment:
I couldn't make heads or tails of this doc comment. Any idea what we were trying to say?

Reply:
This sounds like copy-paste error from the .NET SDK, back when the .NET SDK had a ServiceClient class and an AmqpServiceClient class. I wouldn't worry about deleting it. This java SDK never had that same pattern